samples/meetings-content-bubble/csharp/TargetedNotifications/Bots/TargetedInNotificationMeetingBot.cs

```diff
@@ -0,0 +1,268 @@
+﻿// <copyright file="TargetedInNotificationMeetingBot.cs" company="Microsoft">
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// </copyright>
+
+using AdaptiveCards;
+using AdaptiveCards.Templating;
+using TargetedInNotificationMeetingBot.Models;
+using Microsoft.Bot.Builder;
+using Microsoft.Bot.Builder.Teams;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Bot.Connector;
+using Microsoft.Bot.Schema;
+using Microsoft.Bot.Schema.Teams;
+using Microsoft.Extensions.Configuration;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net.Http.Headers;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web;
+
+
+namespace TargetedInNotificationMeetingBot
+{
+    public class TargetedInNotificationMeetingBot : TeamsActivityHandler
```

Comment:
A minor feedback: Can we update the name to be `TargetedInMeetingNotificationBot` or `TargetedNotificationMeetingBot` as "targeted in meeting notification" / "Targeted Meeting Notification" is the feature name on the public doc?  And let's be consistent with that name throughout the code.

Reply:
Updated the classes at all relevant places